app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    mv_command, file_name, file_path = command_parts
```

Comment:
Unpacking `command_parts` without checking its length can lead to a `ValueError` if the command does not have exactly three parts. Consider adding a check before unpacking to ensure the list has the expected number of elements.